### PR TITLE
PG-99 issue#1

### PR DESCRIPTION
**Pull request by nishant-bhorodia**
This PR addresses the following issue:

**Issue:**
When making a payment on webform civicrm, line items are always linked to Contribution entity rather than membership or participant.

### CODE REVIEW

File: includes/wf_crm_webform_postprocess.inc [hunk 1]
```diff
@@ -1419,7 +1419,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
                 'label' => $this->getMembershipTypeField($type, 'name'),
                 'element' => "civicrm_{$c}_membership_{$n}",
                 // Seems strange to not be civicrm_membership, but this is how it's done in core
-                'entity_table' => 'civicrm_contribution',
+                'entity_table' => 'civicrm_membership',
```
Comment (colemanw) [inline includes/wf_crm_webform_postprocess.inc:1421]: Pls remove the above comment since you have fixed it.

**Comment by nishant-bhorodia** [comment]
Thanks @colemanw . Removed the comment.

File: includes/wf_crm_webform_postprocess.inc (hunk 1 above)
Comment (nishant-bhorodia) [inline includes/wf_crm_webform_postprocess.inc:1421]: Thanks @colemanw . Removed the comment.

**Comment by guanhuan** [comment]
Hi Coleman,

Issue link https://www.drupal.org/node/2553259

Thanks Nishant

Best
Guanhuan

**Pull request merged**